1_fetch/src/download_sb_data.r

```diff
@@ -5,13 +5,12 @@
   library(sbtools)
   library(whisker)
   
-  fetch_data <- function() {
-    project_output_dir <- '1_fetch/out'
+  fetch_data <- function(project_output_dir) {
     #create directory
-    dir.create(project_output_dir)
+    dir.create(project_output_dir, showWarnings = FALSE)
     
     
     # Get the data from ScienceBase
-    mendota_file <- file.path(project_output_dir, 'model_RMSEs.csv')
+    mendota_file <- file.path(project_output_dir)
     item_file_download('5d925066e4b0c4f70d0d0599', names = 'me_RMSE.csv', destinations = mendota_file, overwrite_file = TRUE)
```

Comment:
In our project pipelines, we often want to be able to reuse a function like this to download _any_ file from _any_ Science Base id, rather than just this one specific file. Could you pull out the science base ID and names arguments of `item_file_download()` and include them as arguments to your `fetch_data()` function, both in the function definition and in `_targets.R`? That has the added benefit of allowing any user of the pipeline to see exactly what file is being downloaded from science base while reading through `_targets.R`, rather than having to dig into the `fetch_data()` function to find that information.